pipelines/model-upload/Makefile

```diff
@@ -20,11 +20,11 @@ create:
 
 # upload model to the pod to the PVC
 	@oc exec local-model-to-pvc-pod -- mkdir workspace/${PVC}/model_dir/ -p
-	@oc cp ${MODEL_PATH} local-model-to-pvc-pod:/workspace/${PVC}/model_dir/${NAME}
+	@oc cp ${MODEL_PATH} local-model-to-pvc-pod:/workspace/${PVC}/model_dir-${NAME}/${NAME}
```

Comment:
I wonder whether this is needed as the fix is needed just for the older pipeline, not aiedge-e2e. We can just write in the docs that the location of the uploaded model is changed if somebody wants to run the old pipeline.

FYI the separate build & test pipelines are being deprecated by #198.